stories/topics/proc-gcp-external-load-balancer.adoc

```diff
@@ -51,16 +51,21 @@ For the {HubName} load balancer, the correct instance group has the suffix `-aap
 .. In the resulting dialog box, select *USE EXISTING PORT NAME*.
 .. Set *Balancing mode* to *Rate*.
 .. Set *Maximum RPS* to 300.
+.. Scroll down until you see *Cloud CDN*. Unselect checkbox for *Cloud CDN*.
 .. Scroll down until you see a text box showing *Health check*. 
 .. Select the drop-down menu and click btn:[CREATE A HEALTH CHECK].
 .. Enter a name for your health check, for example, `<DeploymentName>-aap-<cntrlr/hub>-ext-lb-hc`
 .. For {Controllername}, use the following health check settings
-* In the *Health Check* dialog box, set *Protocol* to `TCP`, and *Port* to `8052`.
+* In the *Health Check* dialog box, set *Protocol* to `HTTPS`, and *Port* to `8052`.
 * Set *Request* to `/api/v2/ping/`.
 .. For {HubName}, use the following health check settings
-* In the *Health Check* dialog box, set *Protocol* to `TCP`, and *Port* to `8080`.
+* In the *Health Check* dialog box, set *Protocol* to `HTTPS`, and *Port* to `8080`.
 * Set *Request* to `/api/galaxy/pulp/api/v3/status/`.
-* Keep all other default settings. 
+.. Scroll down to *Health critearia* section.
+.. Set *Check interval* to 15.
```

Comment:
Are these in particular fields?

Reply:
So I guess from:
.. Scroll down to *Health critearia* section.
.. Set *Check interval* to 15.
to:
.. Scroll down to *Health criteria* section.
.. In the *Check interval* field, enter the value 15.